src/quera_ahs_utils/plotting.py

```diff
@@ -27,10 +27,9 @@ def show_register(
 
         Args:
             register (AtomArrangement): A given register
-            blockade_radius (float): The blockade radius for the register. Default is 0
-            what_to_draw (str): Either "bond" or "circle" to indicate the blockade region. 
-                Default is "bond"
-            show_atom_index (bool): Whether showing the indices of the atoms. Default is True
+            blockade_radius (float): Default is 0. The blockade radius for the register.
+            what_to_draw (str): Default is "bond". Either "bond" or "circle" to indicate the blockade region. 
+            show_atom_index (bool): Default is True. Whether showing the indices of the atoms.
```

Comment:
```suggestion
            show_atom_index (bool): Default is True. Choose if each atom's index is displayed over the atom itself in the resulting figure. 
```
Taken from the `plot_avg_density` function docstring, current one isn't grammatically correct.